docs/installation/prepare/microcontrollers/_index.md

```diff
@@ -31,204 +23,88 @@ Viam recommends purchasing the ESP32 with a [development board](https://www.espr
+2. Click on the **Setup** tab.
+3. Regardless of your operating system, select **Mac** and press the button that appears in Step 2 to download the Viam app config for your robot.
+   Copy the path where this <file>viam.json</file> file was downloaded.
 
-#### Install the Rust ESP Toolchain and Activate the ESP-RS Virtual Environment
+   In your terminal, `cd` to the directory where you downloaded your pre-built binary and run the following command to flash the micro-RDK directly to an ESP32 connected to your computer through a data cable:
 
-First, install the following tools with `cargo`:
+   {{< tabs name="Download pre-built binaries" >}}
+   {{% tab name="Linux (x86_64)"%}}
 
 ```sh { class="command-line" data-prompt="$"}
-cargo install espup
+./micro-rdk-installer-amd64-linux write-flash --app-config=<your-file-path-to/viam.json>
 ```
 
-```sh { class="command-line" data-prompt="$"}
-cargo install cargo-espflash v2.0.0-rc.1
-```
+    {{% /tab %}}
+    {{% tab name="Linux (Aarch64)" %}}
 
 ```sh { class="command-line" data-prompt="$"}
-cargo install ldproxy
+./micro-rdk-installer-arm64-linux write-flash --app-config=<your-file-path-to/viam.json>
 ```
 
-To download and install the esp-rs toolchain, run:
+    {{% /tab %}}
+    {{% tab name="MacOS" %}}
 
 ```sh { class="command-line" data-prompt="$"}
-espup install -s -f ~/esp/export-rs.sh -v 1.67.0
+./micro-rdk-installer-macos write-flash --app-config=<your-file-path-to/viam.json>
 ```
 
-To activate the ESP Rust toolchain, run the following command to source (`.`) the activation script `export-rs.sh`:
+    {{% /tab %}}
+    {{% tab name="Windows" %}}
 
 ```sh { class="command-line" data-prompt="$"}
-. $HOME/esp/export-rs.sh
+./micro-rdk-installer-windows write-flash --app-config=<your-file-path-to/viam.json>
 ```
 
-{{< alert title="Tip" color="tip" >}}
-
-To avoid conflicts with other toolchains, adding this command to your .bashrc or .zshrc is not recommended.
-Instead, save this command to run in any future terminal session where you need to activate the ESP-IDF development framework.
+    {{% /tab %}}
+    {{< /tabs >}}
 
-{{< /alert >}}
-
-{{% /tab %}}
-{{% /tabs %}}
+    To see the micro-RDK server logs through the serial connection, add `--monitor`.
+    If the program cannot auto-detect the serial port to which your ESP32 is connected, you may be prompted to select the correct one among a list.
 
-## Install the Micro-RDK
-
-### Create a New Robot
-
-Navigate to [the Viam app](https://app.viam.com) and [add a new robot](/manage/fleet/robots/#add-a-new-robot) in your desired location.
+Go back to your new robot's page on [the Viam app](https://app.viam.com).
+If successful, **Live** should be displayed underneath **Last online**.
 
-Click on the name of the robot to go to the robot's page.
-Then, navigate to the **Config** tab.
+For more `micro-rdk-installer` CLI usage options, see [GitHub](https://github.com/viamrobotics/micro-rdk/tree/main/micro-rdk-installer).
```

Comment:
```suggestion
For more `micro-rdk-installer` CLI usage options, see [GitHub](https://github.com/viamrobotics/micro-rdk/tree/main/micro-rdk-installer).
```

@gvaradarajan Here I could also document how to write Wi-fi credentials, but I'm not sure what the command for that would look like exactly from readme after `./micro-rdk-installer-amd64-linux write-credentials`. Do you want to document this on docs.viam, and if so do you have an example of providing credentials in a command that I could use?

Reply:
Ok, sounds good to me!